src/web/package.json

```diff
@@ -17,22 +17,17 @@
     "dotenv": "^8.2.0",
     "highlight.js": "10.7.2",
     "jwt-decode": "^3.1.2",
-    "material-ui-popup-state": "^1.8.0",
     "nanoid": "^3.1.22",
     "next": "^10.1.3",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "react-material-ui-form-validator": "^2.1.4",
     "react-use": "^17.2.1",
-    "swr": "^0.5.5",
-    "valid-url": "^1.0.9"
+    "swr": "^0.5.5"
   },
   "devDependencies": {
     "@types/node": "^14.14.37",
     "@types/react": "^17.0.3",
-    "@types/react-material-ui-form-validator": "^2.1.0",
     "@types/react-test-renderer": "^17.0.1",
-    "@types/valid-url": "^1.0.3",
```

Comment:
All of these dependencies being removed were being used only on`/myfeeds` ?

Reply:
Yeah, I believe so.  If we need them in new code, add them back, or flag now and I can leave them in.

Reply:
Okay. let's remove them.